tests/coalaCITest.py

```diff
@@ -105,6 +105,9 @@ def test_coala_delete_orig(self):
              make_temp(suffix='.coafile', prefix='', dir=tempdir) as coafile,\
              make_temp(dir=tempdir) as unrelated_file:
             orig_file.close()
+            with open(coafile, mode="w") as file:
+                file.write("files = something\n")
+
```

Comment:
it's needed as the new function looks for where the files setting is defined, not which config file is used (so if the files setting comes from CLI the cwd is the project dir and not the dir of the config file which might be somewhere else)